From: LimaLinux
Subject: lang (es): botón “Go back” sin % (Regresar)
---
diff --git a/lang/calamares_es.ts b/lang/calamares_es.ts
--- a/lang/calamares_es.ts
+++ b/lang/calamares_es.ts
@@ -517,7 +517,7 @@
       <location filename="../src/libcalamaresui/ViewManager.cpp" line="355"/>
       <source>Go &amp;Back</source>
       <comment>@button</comment>
-      <translation>%Regresar</translation>
+      <translation>Regresar</translation>
     </message>
     <message>
       <location filename="../src/libcalamaresui/ViewManager.cpp" line="402"/>
